index.js

```diff
@@ -107,6 +112,30 @@ app.delete('/ideas', (req, res) => {
     res.status(200).send('Ideas successfully deleted.');
 });
 
+//DONATION METHODS
+
+app.get('/donations', (req, res) => {
+    res.status(200).json(ideasDonationSum(req.body.ideaId));
+});
+
+app.get('/donations/sumDifference', (req, res) => {
+    res.status(200).json(ideasSumDifference(req.body.ideaId));
+});
+
+app.post('/donations', (req, res) => {
+    try {
+        createDonationsWithDataValidation(
+            req.body.firstName,
+            req.body.sum,
+            req.body.userId,
+            req.body.ideaId
+        );
+    } catch (Error) {
```

Comment:
Should be `error`.

`Error` is a global JavaScript function.

Reply:
updated and commited